project2/src/store/Slice/ProductSlice.jsx

```diff
@@ -0,0 +1,47 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
```

Comment:
Tên file đặt như này em nhé
`store/slice/productSlice.js`
Đặt tên in hoa hoàn toàn chỉ áp dụng cho các Class Component, hoặc Functional Component.
Đuôi jsx cũng tương tự, những thứ k handle, k liên quan gì tới code HTML thì chỉ là .js thôi